doc/api/tools_variable_substitutions.md

```diff
@@ -476,6 +476,13 @@ returns the current course's term start date.
 
 **Availability**: *when launched in a course that has a term with a start date*  
 
+```
+active
```

Comment:
I think this is supposed to be an example of the previous variable substitution, so `YYY-MM-DD HH:MM:SS -0700` (from new line 488) makes more sense here

Reply:
See reply to comment below.

Reply:
Fix docs (sorry about this, did a quick+bad copy paste and didn't think to look for auto generated docs)